readme.md

```diff
@@ -18,6 +18,9 @@ var isAbsoluteUrl = require('is-absolute-url');
 isAbsoluteUrl('http://sindresorhus.com/foo/bar');
 //=> true
 
+isAbsoluteUrl('//sindresorhus.com');
+//=> false
```

Comment:
Not sure myself. Would an explicit note add more value than the example? Or just to emphasise it?